package.json

```diff
@@ -30,25 +30,37 @@
   },
   "homepage": "https://github.com/fastify/fastify-static",
   "dependencies": {
-    "fastify-plugin": "^1.6.0",
+    "fastify-plugin": "^2.0.0",
     "glob": "^7.1.4",
     "readable-stream": "^3.4.0",
     "send": "^0.16.0"
   },
   "devDependencies": {
     "@types/node": "^10.14.9",
+    "@typescript-eslint/eslint-plugin": "^2.29.0",
+    "@typescript-eslint/parser": "^2.29.0",
     "concat-stream": "^2.0.0",
     "coveralls": "^3.0.4",
     "eslint-plugin-typescript": "^0.14.0",
-    "fastify": "^2.5.0",
-    "fastify-compress": "^0.8.1",
+    "fastify": "^3.0.0-alpha.1",
+    "fastify-compress": "github:thomheymann/fastify-compress#v3-upgrade",
     "pre-commit": "^1.2.2",
     "proxyquire": "^2.1.0",
     "simple-get": "^3.0.3",
     "snazzy": "^8.0.0",
     "standard": "^12.0.0",
+    "standardx": "^5.0.0",
```

Comment:
How does this differ from `standard`? Should we remove `standard` if we are using this module instead?

Reply:
`standardx` is standard with some custom tweaks and it is needed beacuse I removed the `typescript-eslint-parser` because it was deprecated in favour of `typescript-eslint`. But that errors for interface imports. Based on standards [docs](https://standardjs.com/#typescript) using `standardx` is the workaround. However is not as frequently updated as `standard` and that's why I would keep using it for default linting and use `standardx` only for `typescript`

Reply:
Okay, I don't think we are linting ts files in other modules. I usually use eslint and then include the the typescript and standard plugins with it, but doesn't matter. If this works its good with me 😄

Reply:
I'm not really ok in adding this :/. What do we use for the other plugins?

Reply:
This is the only project that includes linting for the types test file.

Reply:
I would just drop linting for the types or move to eslint.